packages/discord.js/src/client/actions/Action.js

```diff
@@ -25,7 +25,7 @@ class GenericAction {
 
   getPayload(data, manager, id, partialType, cache) {
     const existing = manager.cache.get(id);
-    if (!existing && this.client.options.partials.includes(partialType)) {
+    if (this.client.options.partials.includes(partialType)) {
       return manager._add(data, cache);
     }
     return existing;
```

Comment:
Without the `existing` check, the line 27 is used only below, it can be inlined, although we can also make it a ternary:
```javascript
return this.client.options.partials.includes(partialType)
  ? manager._add(data, cache)
  : manager.cache.get(id);
```

Reply:
Resolved with 88b9dcc877e09ddc5c81fbe4122eb97f39145fed.